ext/standard/pack.c

```diff
@@ -104,8 +104,47 @@ static void php_pack(zval *val, size_t size, int *map, char *output)
 }
 /* }}} */
 
+
+/* {{{ php_pack_is_little_endian
+ */
+static int php_pack_is_little_endian()
+{
+	int machine_endian_check = 1;
+	
+	return ((char *)&machine_endian_check)[0];
```

Comment:
If they would not match, you could not use bit-tricks on floats with integers... (which we anyway do)